### PR TITLE
Release 0.37.0

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
Release v0.37.0

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by phoracek** [comment]
@qinqon would you please confirm this release works on OpenShift too?

File: manifests/cluster-network-addons/0.37.0/cluster-network-addons-operator.0.37.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,209 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: cluster-network-addons-operator.0.37.0
+  namespace: placeholder
```
Comment (RamLavi) [inline manifests/cluster-network-addons/0.37.0/cluster-network-addons-operator.0.37.0.clusterserviceversion.yaml:5]: is this right?

Comment (phoracek) [reply manifests/cluster-network-addons/0.37.0/cluster-network-addons-operator.0.37.0.clusterserviceversion.yaml:5]: Yeah, OLM uses it AFAIK

**Comment by qinqon** [comment]
/hold

Looks like we have issues running kubemacpool e2e tests on deployed CNAO at k8s.

**Pull request closed without merging**